src/ErrorHandler.php

```diff
@@ -137,6 +143,10 @@ public static function registerOnce(int $reservedMemorySize = self::DEFAULT_RESE
             @trigger_error(sprintf('Method %s() is deprecated since version 2.1 and will be removed in 3.0. Please use the registerOnceErrorHandler(), registerOnceFatalErrorHandler() or registerOnceExceptionHandler() methods instead.', __METHOD__), E_USER_DEPRECATED);
         }
 
+        if (null === self::$handlerInstance) {
+            self::$handlerInstance = new self();
```

Comment:
The problem was that Psalm was not able to infer from the subsequent calls that the property could never be `null` before the `return, so I had to choose between ignoring the error or add this code. I decided to avoid ignoring errors as much as I could because doing it feels like cheating